Source/Charts/Utils/Platform+Accessibility.swift

```diff
@@ -33,7 +33,7 @@ open class NSUIAccessibilityElement: UIAccessibilityElement
         }
     }
 
-    override init(accessibilityContainer container: Any)
+    override public init(accessibilityContainer container: Any)
```

Comment:
shall we change it to open? @petester42 @jjatie

Reply:
yes!

Reply:
Yep!